apps/api/src/app/topics-v2/dtos/create-topic-subscriptions-response.dto.ts

```diff
@@ -0,0 +1,173 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ISubscriptionData, ISubscriptionError, ITopicSubscriptionResult } from './subscription-interfaces';
+
+export class TopicDto {
+  @ApiProperty({
+    description: 'The unique identifier of the topic',
+    example: '64f5e95d3d7946d80d0cb677',
+  })
+  _id: string;
+
+  @ApiProperty({
+    description: 'The key identifier of the topic',
+    example: 'product-updates',
+  })
+  key: string;
```

Comment:
i wonder if in those cases we need to explain that one is novu internal and one is an external key.
in addition the `key` should be unique as well.